docker-compose.yml

```diff
@@ -1,17 +1,20 @@
-web:
-  image: colinxfleming/dcaf_case_management
-  command: rails s -p 3000 -b 0.0.0.0
-  volumes:
-    - .:/usr/src/app
-  ports:
-    - "3000:3000"
-  links:
-    - db
-  environment:
-    mongohost: db
-  env_file:
-    - web-variables.env
-db:
-  image: mongo
-  ports:
-    - "27017:27017"
+version: '3.0'
```

Comment:
Well, without this change, `docker-compose` won't allow you to build the image -- and we need to rebuild it to pick up the Dockerfile change.

It looks like we could go with `docker-compose` version 2 though, which supports specifying build + image too -- but then we have to do some other tweaking to apply a CircleCI patch to allow Docker 1.10?